axelrod/strategies/axelrod_second.py

```diff
@@ -199,3 +202,214 @@ def strategy(self, opponent: Player) -> Action:
         else:
             # Play TFT
             return opponent.history[-1]
+
+class Tranquilizer(Player):
+   
+    """
+    Submitted to Axelrod's second tournament by Craig Feathers
+
+    This strategy is based on the reverse engineering of the 
+    Fortran strategy K67R from Axelrod's second tournament.
+    Reversed engineered by: Owen Campbell, Will Guo and Mansour Hakem.
+
+    The strategy starts by cooperating this strategy has 3 states.
+
+    At the start of the strategy it updates it's states:
+
+    - It counts the number of consecutive defections by the opponent.
+    - If it was in state 2 it moves to state 0 and calculates the 
+    following quantities two_turns_after_good_defection_ratio and
```

Comment:
These need two spaces:

```
- If it was ...
  following...
  two_turns_...
```

also though include the formulae for `two_turns_after_good_defection_ratio` and `two_turns_after_good_defection_ratio_count`.